src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionResolveTests.cs

```diff
@@ -376,23 +373,23 @@ private static async Task<LSP.CompletionItem> RunResolveCompletionItemAsync(Test
         private static ClassifiedTextRun[] CreateClassifiedTextRunForClass(string className)
             => new ClassifiedTextRun[]
             {
+                new ClassifiedTextRun("whitespace", string.Empty),
                 new ClassifiedTextRun("keyword", "class"),
                 new ClassifiedTextRun("whitespace", " "),
-                new ClassifiedTextRun("class name", className)
+                new ClassifiedTextRun("class name", className),
+                new ClassifiedTextRun("whitespace", string.Empty),
```

Comment:
There's behavior changes happening here?

Reply:
This is caused by actually doing serialization - the client extension converters add the whitespace classified text runs